nodejs/eks/cluster.ts

```diff
@@ -260,7 +262,9 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     }
 
     // Create the VPC CNI management resource.
-    const vpcCni = new VpcCni(`${name}-vpc-cni`, kubeconfig, args.vpcCniOptions, { parent: parent });
+    const yamlPath = path.join(__dirname, "cni", "aws-k8s-cni.yaml");
+    const cniYamlText = fs.readFileSync(yamlPath).toString();
+    const vpcCni = new VpcCni(`${name}-vpc-cni`, cniYamlText, kubeconfig, args.vpcCniOptions, { parent: parent });
```

Comment:
This looks fine as-is - but FWIW - I had been imagining a slightly more tactical change that kept this inside the `VpcCni` constructor, just not inside the `applyVpcCniYaml` implementation (which is run inside a different process).

Reply:
I'll work on a change that has it in the `VpcCni` constructor.